lib/deps.ts

```diff
@@ -80,14 +85,14 @@ function loadModulesInternal(root, rootDepType, parent, options) {
     return Promise.reject(new Error('module path must be a string'));
   }
 
-  var modules = {};
+  var modules = {} as PackageExpanded;
```

Comment:
change vars to `let/const`?

Reply:
Intentionally omitted to keep the diff short.